python/lsst/ts/Dome/mock_controller.py

```diff
@@ -261,16 +264,16 @@ async def move_az(self, **kwargs):
         Parameters
         ----------
         kwargs: `dict`
-            A dictionary with arguments to the function call. It should contain the key "azimuth" with a
-            float value between 0 and 2pi and the key "azRate" with a float value where positive means
+            A dictionary with arguments to the function call. It should contain the key "position" with a
+            float value between 0 and 2pi and the key "velocity" with a float value where positive means
```

Comment:
For this and all other commands please list the arguments directly instead of using `kwargs`. In other words:
```
async def move_az(self, position, velocity):
    """Move the dome.

    Parameters
    -----------
    position : `float`
        Desired azimuth, in radians, in range [0, 2 pi)
    ...
    """
```
It makes the API much clearer and easier to understand and catches mistakes in spelling the arguments immediately.

The only thing it makes harder is logging the info, and I personally think a clear API is much more important than simplicity of logging.